webui/package.json

```diff
@@ -6,7 +6,7 @@
   },
   "scripts": {
     "dev": "vite",
-    "build": "vite build",
+    "build": "cross-env NODE_OPTIONS=--max-old-space-size=4096 vite build",
```

Comment:
Nice - didn't used cross-env before (don't like extra packages). My thought was adding it to the Makefile so no specific value is forced into my build. or in my case when I'm trying 'bun' to build the code.

Reply:
@nopcoder I actually think the npm scripts is a great place to keep build-related node params. I think it's the right place for them. This is so inconsequential that I'm starting to feel a bit silly going on about it 😊